src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new NoSuchElementException("Movie not found."));
```

Comment:
The exception message in the `get` method should be more descriptive. Consider including the `id` in the message to provide more context when a movie is not found.